aiutils/data/data.py

```diff
@@ -0,0 +1,87 @@
+"""Utility functions for making batch generation easier."""
+import numpy as np
+from multiprocessing import Process, Queue
+
+
+def sequential(batch_size, num_samples, num_epochs=1):
```

Comment:
Good point. It should be more specific. How about `sequential_generator` and `random_generator`?